temporal-sdk/src/test/java/io/temporal/workflow/WorkflowTest.java

```diff
@@ -4264,6 +4267,10 @@ public String execute(String testName) {
 
   @Test
   public void testWorkflowWithCronSchedule() {
+    // REVIEW: Min interval can be less than 1min when running against a real server - what explains
+    // that discrepancy?
+    //   Why doesn't "@every 1s" work here? Different cron parsing libs?
```

Comment:
We need to update the documentation. The discrepancy is because the service uses https://github.com/robfig/cron and the java unit test service uses https://github.com/jmrozanec/cron-utils.

The time skipping is the major feature of the unit testing framework that supports unit testing long running workflows.

Reply:
Thanks for the answer! Can you point me at info/code on the time skipping stuff?